test/infer/test_autoguide.py

```diff
@@ -183,7 +183,8 @@ def test_uniform_normal():
     def model(data):
         alpha = numpyro.sample('alpha', dist.Uniform(0, 1))
         with numpyro.handlers.reparam(config={'loc': TransformReparam()}):
-            loc = numpyro.sample('loc', dist.Uniform(0, alpha))
+            loc = numpyro.sample('loc', dist.TransformedDistribution(
+                dist.Uniform(0, 1), transforms.AffineTransform(0, alpha)))
```

Comment:
Will this break existing users?  If so, will the breakage emit an informative error message?

Reply:
Yes. I'll raise a more informative message in `TransformReparam`.